modules/dxp/apps/portal-search-tuning/portal-search-tuning-rankings-web/src/main/resources/META-INF/resources/view_results_rankings.jsp

```diff
@@ -112,6 +114,17 @@ RankingPortletDisplayContext rankingPortletDisplayContext = (RankingPortletDispl
 				value="<%= rankingEntryDisplayContext.getHiddenResultsCount() %>"
 			/>
 
+			<liferay-ui:search-container-column-text
+				cssClass="table-cell-expand-smallest table-cell-minw-150"
+				name="status"
+			>
+				<div class="label label-success">
```

Comment:
`label-success` should be conditional.

Maybe something like:
`<%= rankingEntryDisplayContext.getStatus() == ResultRankingsContants.STATUS_ACTIVE ? "label-success" : "" %>`